Dockerfile-linux.template

```
@@ -221,8 +241,20 @@ RUN set -eux; \
 		gpgconf --kill all; \
 		rm -rf "$GNUPGHOME"; \
 	fi; \
+{{ if .version == "8.4.0alpha1" then ( -}}
+	\
+	# Add patch; see https://github.com/docker-library/php/pull/1526
+	curl -fsSL -o php-pdo.patch https://patch-diff.githubusercontent.com/raw/php/php-src/pull/14797.patch; \
+	echo "d9f33beda1ffffc66299377a0946b1fe2916382e320084568ae6ee2a6d1fb19e php-pdo.patch" | sha256sum -c -; \
+	TEMPFILE=$(mktemp); \
+	filterdiff -p1 -x 'NEWS' < php-pdo.patch >$TEMPFILE; \
+	mv $TEMPFILE php-pdo.patch; \
```

TIL `patchutils`!  How have I not known of this before? :sweat_smile: :heart:

Minor nit, but I think we should do this with just a `.xxx` or similar instead of a whole `mktemp`:

```suggestion
	filterdiff -p1 -x 'NEWS' < php-pdo.patch > php-pdo.patch.filtered; \
	mv php-pdo.patch.filtered php-pdo.patch; \
```